packages/eas-cli/src/run/android/aapt.ts

```diff
@@ -0,0 +1,58 @@
+import spawnAsync, { SpawnResult } from '@expo/spawn-async';
+import glob from 'fast-glob';
+import unixify from 'unixify';
+
+import Log from '../../log';
+import { getAndroidSdkRootAsync } from './sdk';
+
+async function aaptAsync(...options: string[]): Promise<SpawnResult> {
+  try {
+    return await spawnAsync(await getAaptExecutableAsync(), options);
+  } catch (error: any) {
+    if (error.stderr) {
+      Log.error(error.stderr);
+    }
+    throw error;
+  }
+}
+
+export async function getAaptExecutableAsync(): Promise<string> {
+  const sdkRoot = await getAndroidSdkRootAsync();
+  if (!sdkRoot) {
+    Log.debug('Failed to resolve the Android SDK path, falling back to global aapt executable');
+    return 'aapt';
+  }
+  const aaptPaths = await glob(
+    `${unixify(sdkRoot)}/build-tools/**/${
+      ['win32', 'win64'].includes(process.platform) ? 'aapt.exe' : 'aapt'
+    }`
+  );
```

Comment:
After fixing this part I was able to successfully run the command on my Windows machine 🎉

Reply:
I will add few UX improvements for Windows users and I think than it should be good to go 😄

Reply:
where did you get win64 from? there is no such platform, also you did not address Dominik's comment about using path.join instead of interpolation

Reply:
@wkozyra95 I thought `win64` is a thing, so sorry 🤦.  It should be fixed now.